### PR TITLE
Merge dev-zh into main(for Chinese localization live version)

### PR DESCRIPTION
**Pull request by Jacob953**
Merge Chinese development branch into main.

Checklist to make localization live.

 - [x] validate all files which are not in the localization directory (content/zh/*)
 - [x] check i18n/zh.toml exist 
 - [x] acknowledge to initial localization contributors list #869 
 - [x] build project of [Chinese Location Progress](https://github.com/cncf/glossary/projects/8)
 - [x] check Preview works
 - [x] check l10n configuration in config.toml #843
 - [x] update contributor-ladder to localization #874

### CODE REVIEW

File: content/zh/_index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+对本机词汇表的更改、添加和改进对所有人开放. 我们使用由 CNCF 管理的社区驱动流程来开发和完善这个共享词典. 该词汇表提供了一个厂商中立的平台, 用于管理云原生技术的共享词汇表. 欢迎所有参与者根据本项目的目的和章程作出贡献.
+
+任何想要贡献的人都可以提交 GitHub issue 或 创建 PR. 欲了解更多信息, 请查看[如何贡献](/zh/contribute/), 并且必须遵循[样式指南](/zh/style-guide/).
+
+## 致谢
+
+云原生词汇表可从 CNCF 营销委员会 (CNCF Marketing Committee) (商业价值小组委员会) 获得,
+包括 [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/),
+[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/) 以及更多的贡献者. 如想获得完整的贡献者名单, 请跳转至 [GitHub Pages 页](https://github.com/cncf/glossary/graphs/contributors).
+
```
Comment (seokho-son) [inline content/zh/_index.md:21]: Hello, @Jacob953 
Please add acknowledgement to localization contributors to _index.md. 
(similar style with ack to Glossary contributors)

You can open another PR to add it.

Comment (Jacob953) [reply content/zh/_index.md:21]: Thx for the reminder! @seokho-son 
I've handled this in #869 ;)

File: content/zh/contribute/_index.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+![submit issue](/images/how-to/howto-15.png)
+
+### 直接更新术语
+如要直接更改术语, 请转到“编辑此页面”.
+
+![edit this page](/images/how-to/howto-16.png)
+
+这将打开术语的 GitHub 页面. 进行更改并提交 PR. 有关详细说明, 请参阅上面的“提交新术语”(跳转到有关降价的部分).
+
+## Help translate the glossary
```
Comment (jihoon-seo) [inline content/zh/contribute/_index.md:106]: Please translate this header into Chinese. 😊

File: content/zh/style-guide/_index.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+---
+title: 风格指南
+toc_hide: true
+menu:
+  main:
+    weight: 10 
+    pre: <i class='fas fa-ruler-horizontal'></i>
+---
+
+The following style guide is designed to help you understand the glossary definition structure and maintain a consistent style throughout.
```
Comment (jihoon-seo) [inline content/zh/style-guide/_index.md:10]: Please translate this "Style Guide" page into Chinese. 😊

**Review by jihoon-seo — COMMENTED** [review]
@Jacob953 PTAL!

**Comment by jihoon-seo** [comment]
@Jacob953 In preview, the Chinese version of ["Contributor Ladder" page](https://deploy-preview-844--cncfglossary.netlify.app/contributor-ladder/) is missing.
Could you (or anyone in Chinese L10n team) add this? 😊

File: content/zh-cn/_index.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## 致谢
+
+云原生词汇表可从 CNCF 营销委员会 (CNCF Marketing Committee) (商业价值小组委员会) 获得,
+包括 [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/),
+[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/) 以及更多的贡献者. 如想获得完整的贡献者名单, 请跳转至 [GitHub Pages 页](https://github.com/cncf/glossary/graphs/contributors).
+
+中文版的云原生词汇表由 [Chester Cheung](https://github.com/hanyuancheung), 
+[Brian Yan](https://github.com/Rocksnake), 
+[Jacob953 Yu](https://github.com/Jacob953),
```
Comment (seokho-son) [inline content/zh-cn/_index.md:26]:
```suggestion
[Junfeng Yu](https://github.com/Jacob953), 
```
@Jacob953 just in case it is a typo ;)

Comment (Jacob953) [reply content/zh-cn/_index.md:26]: Thx for the reminder, I usually surf the Internet by the alias Jacob953 Yu. ;)

**Review by seokho-son — APPROVED** [review]
L!G!T!M! 

Thanks to the Chinese l10n team !!

**Comment by jihoon-seo** [comment]
FYI: The Chinese version of "How To Contribute" seems slightly outdated. 😊

- zh: https://deploy-preview-844--cncfglossary.netlify.app/zh-cn/contribute/
- en: https://deploy-preview-844--cncfglossary.netlify.app/contribute/

**Comment by hanyuancheung** [comment]
> FYI: The Chinese version of "How To Contribute" seems slightly outdated. 😊
> 
> * zh: https://deploy-preview-844--cncfglossary.netlify.app/zh-cn/contribute/
> * en: https://deploy-preview-844--cncfglossary.netlify.app/contribute/

@Jacob953 yeah, pls update the content mentioned above, which is aligned with English version.

**Review by jihoon-seo — APPROVED** [review]
LGTM 😊

**Review by hanyuancheung — APPROVED** [review]
lgtm

**Comment by Jacob953** [comment]
Hi, @jihoon-seo PTAL!
I've update contribute/_index.md & style-guide/_index.md to dev-zh with main in #881 

I'll merge it into main as soon as #881 is done. ;)

**Pull request merged**